src/components/relationships.tsx

```diff
@@ -47,7 +48,60 @@ export const RelationshipsTab = (
 ) => {
   const { datasource, query, onQueryChange, queryBinder } = props;
   const route = queryBinder ? `${queryBinder}.${queryTypeSelector}` : `${queryTypeSelector}`;
-
+  const [options, setOptions] = useState([]);
+  const getOptions = async () => {
```

Comment:
Recommendation: Lets rename this to `updateOptions`, since this doesnt really `get` anything